### ISSUE TITLE
DfE content style guide

### ISSUE THREAD

**Issue by kerryalyons** [DFE-Digital/design#111]
Fix typo in intro:

Add 'from' to - 'This style guide includes words that differ the GDS style guide'

**Comment by adyhoran1** [comment]
Could you include:

- links in the L section? I regularly refer to the [2.2.4 WCAG criterion](https://design.education.gov.uk/accessibility/understanding-wcag/operable#2-4-4) when writing link text, but the criterion is not the easiest to find. I feel it could be a useful addition to the DfE style guide.
- subject matter expert in the S section? That term is used by all[ Regional Services Division (RSD)](https://design-histories.education.gov.uk/team/regional-services-division) teams, and I'd imagine that other DfE teams use this term as well
- the [acronyms guidance from Regional Services Division (RSD)](https://educationgovuk.sharepoint.com/:w:/r/sites/ServiceDeliveryDirectorate/Shared%20Documents/Content%20design/RSD%20style%20guide.docx?d=w5cad5d1c92c04425ab577b9d032cee1a&csf=1&web=1&e=tJaAJN)? Particularly the 'Acronyms in a series of pages' section

Could you also remove the extra space after 'local council' in this sentence: The GDS style guide uses **[local council ]**(https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#local-council).

**Comment by adyhoran1** [comment]
Could you include Links in the L section? I regularly refer to the [2.2.4 WCAG criterion](https://design.education.gov.uk/accessibility/understanding-wcag/operable#2-4-4) when writing link text, but the criterion is not the easiest to find. I feel it could be a useful addition to the DfE style guide.

**Comment by SteveMilnes** [comment]
Remove `Case owner` and `Project owner` from the guide in the section about `job titles`. 

This conflicts with our guidance to avoid the job title as a descriptor. It's not a job title, but it's the kind of description that will become a way of naming a role that will at some point become outdated. 

This guidance is taken from the RSD style guide, however we will be removing mentions of Project owner and replacing them with `Assigned to` in the products that make up the Manage academy projects team soon.

We should also remove the examples of `Prepared for advisory board by` and `Completed by` as they are not in use. Our SME has told us that `Assigned to` is what users understand most clearly and is the language that is widely used.

**Comment by SteveMilnes** [comment]
> Could you include:
> 
> * links in the L section? I regularly refer to the [2.2.4 WCAG criterion](https://design.education.gov.uk/accessibility/understanding-wcag/operable#2-4-4) when writing link text, but the criterion is not the easiest to find. I feel it could be a useful addition to the DfE style guide.
> * subject matter expert in the S section? That term is used by all[ Regional Services Division (RSD)](https://design-histories.education.gov.uk/team/regional-services-division) teams, and I'd imagine that other DfE teams use this term as well
> * the [acronyms guidance from Regional Services Division (RSD)](https://educationgovuk.sharepoint.com/:w:/r/sites/ServiceDeliveryDirectorate/Shared%20Documents/Content%20design/RSD%20style%20guide.docx?d=w5cad5d1c92c04425ab577b9d032cee1a&csf=1&web=1&e=tJaAJN)? Particularly the 'Acronyms in a series of pages' section
> 
> Could you also remove the extra space after 'local council' in this sentence: The GDS style guide uses **[local council ]**(https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#local-council).

Hi @adyhoran1 

Thanks for your suggestions. We've spoken about them as a group and decided:

- there is guidance for how to style and write links in the GOV.UK style guide and we should not duplicate that
- SME feels like a good addition to a glossary so we'll pick that up as part of that work
- we'll add a sentence to the acronyms guidance in the DfE style guide to help explain that you do not need to write the acronym's full name out on each page if it is part of a journey that takes place in 1 sitting. Like, completing a task on a task list.

We will remove the extra space from local council.

**Comment by Sarah-DfE** [comment]
To add to style guide:

'SENCo' - We follow the GOV.UK style guide for SEN and SEND and apply it to SENCo (special educational needs co-ordinator) and SENDCo (special educational needs and disabilities co-ordinator).

**Comment by SteveMilnes** [comment]
@adyhoran1 @Sarah-DfE I've made the PR for those content changes. Should appear in the style guide later today.

**Comment by frankieroberto** [comment]
One term to consider for the style guide:

"Maths" v "Mathematics" (or "it depends").

**Comment by SteveMilnes** [comment]
Daniel Deakin has suggested some changes based on GOV.UK style.

- delete ‘the’ before EFSA - we don't use the before acronyms
- delete ‘date’ section - we follow GOV.UK style guide
- capitalisation - we follow GOV.UK style guide
- Add ‘advice’ ‘we provide guidance, not advice’
- Add ‘Please’ - we do not use (with a few lines on why - [here is some GDS guidance](https://www.gov.uk/service-manual/design/writing-for-user-interfaces#tone)
- Opening sentence - be more explicit ‘We follow the GDS style guide’ (something like that)

**Comment by adyhoran1** [comment]
Suggestions of things to include:

1.  [Abbreviations and acronyms - Readability Guidelines](https://readabilityguidelines.co.uk/grammar-points/abbreviations-and-acronyms/). To emphasise that abbreviations and acronyms make text harder to read.

2. If writing acronyms out in-full would make the h1 too long. In this case, you could use the acronym in the H1 and put the full explanation later on in the page. Ideally in the first line or as early as possible in the main body copy 

3. Write acronyms in capital letters. This is implied in the GDS style guide, but we should be explicit about this in the DfE guide. Our assumption is that could help some neurodivergent colleagues who might not pick up the implication

4. Use title case for division and team names. For example: 

Regions Group 
Regional Services Division 
Regional Casework Services 
Record Concerns and Support for Trusts 
Manage Academy Projects 

5. use sentence case for document names. If the document name starts a sentence, the first letter should be uppercase. If it’s mentioned within a sentence, it should all be lower case. For example: 

Articles of association 
Make sure the articles of association have been signed by the Secretary of State 

6. Regional Director. Should this be upper case because it's an official title, or lower case because there are 9 regional directors?

**Comment by adyhoran1** [comment]
Suggested terms to add:

- academy. Write it in lower case, unless it's part of an academy's name. For example, John Beckett Academy **Note** - academy is also mentioned in the [DfE writing style guide on the intranet](https://educationgovuk.sharepoint.com/sites/communications/SitePages/style-guide.aspx#4.-capital-letters)
- free school. Write it in lower case
- trust. Write it in lower case, unless it's part of an academy's name. For example, Bright Beginnings Trust

In the job titles section, could you also link to the [job titles section of the GOV.UK style guide](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#job-titles)?

**Comment by SteveMilnes** [comment]
We should edit the example in the 'job titles' section. It's a bit shonky.

> We try to avoid generic job titles because they can change regularly. We don't use the job title as an identifier alongside a name, for example:
> 
> Delivery officer: Joe Bloggs
> Caseworker: Francis Blunderbuss
> 
> We describe the thing that the job role does. This should make more sense to more people and help guard against future changes, for example:
> 
> Prepared for advisory board by: Joe Bloggs
> Completed by: Francis Blunderbuss
> Case owner
> Project owner"

We should probably say something like this instead:

> We try to avoid generic job titles because they can mean different things to different people. 
> 
> For example, a school business manager might have different responsibilities from one school to another.
> 
> Instead, use language that describes the thing a job role does. Something like, "The person who manages your recruitment should upload the job advert" rather than "Your HR Lead should upload the job advert".
> 
> For internal products and services, try not to use the job title as an identifier alongside a name, for example:
> 
> Delivery officer: Joseph Bloggs
> Caseworker: Josephine Bloggs
> Project lead: Joey Bloggs
> 
> We would write that a project is assigned to a particular person, for example:
> 
> Assigned to: Josephine Bloggs
> 
> Job titles can and do change as departments and divisions reorganise and restructure. Avoiding job titles can help to reduce confusion when those titles are no longer used.

**Comment by kerryalyons** [comment]
I will update this.

**Comment by dhomerDFE** [comment]
I noticed that the 'buy or buying' entry explains that this the correct term to use before saying which ones should be avoided.

Would it be better for corrections to be under the word that the reader might otherwise think is correct? The idea being that if I already think I should be using the term 'buy or buying', it's unlikely I'd be looking there to find out which words it shouldn't be? 

The GDS style guide used to do something like "procurement - use buy or buying instead". That looks like it's changed now though, so I appreciate there may be a newer school of thought that I'm not aware of.

**Comment by SteveMilnes** [comment]
> I noticed that the 'buy or buying' entry explains that this the correct term to use before saying which ones should be avoided.
> 
> Would it be better for corrections to be under the word that the reader might otherwise think is correct? The idea being that if I already think I should be using the term 'buy or buying', it's unlikely I'd be looking there to find out which words it shouldn't be?
> 
> The GDS style guide used to do something like "procurement - use buy or buying instead". That looks like it's changed now though, so I appreciate there may be a newer school of thought that I'm not aware of.

I think that's a really good point @dhomerDFE! We should do that.
